tests/roots/test-ext-viewcode-find-package/main_package/__init__.py

```diff
@@ -0,0 +1 @@
+import main_package.subpackage as subpackage  # NOQA
```

Comment:
don't use blanket `NOQA`s. I'm guessing this is an F401?

Reply:
@danieleades I was just following the other viewcode tests as examples:


```
from .mod1 import Class1, func1  # NOQA
from .mod2 import Class2, func2  # NOQA
```

https://github.com/sphinx-doc/sphinx/blob/5.x/tests/roots/test-ext-viewcode/spam/__init__.py


```
from .submodule import Class1, func1  # NOQA
```

https://github.com/sphinx-doc/sphinx/blob/5.x/tests/roots/test-ext-viewcode-find/not_a_package/__init__.py


But I can change all of them if you want me to.

Reply:
please. a blanket noqa may inadvertently squash other lints. Lint suppression should be as targeted as possible.

Reply:
@danieleades Okay, I've replaced all 4 instances of `# NOQA` with `# noqa: F401`.